Cargo.toml

```diff
@@ -13,6 +13,9 @@ documentation = "https://docs.rs/rust-htslib"
 
 [dependencies]
 libc = "0.2"
+libz-sys = "1.0"
+bzip2-sys = { version = "0.1.6", git = "https://github.com/alexcrichton/bzip2-rs.git", optional = true }
```

Comment:
Not sure I understand this. I thought these sys crates are just the bindings, but we don't need bindings here do we?

Reply:
Do they also ship the libs?

Reply:
libz-sys is here